test/test_utils/__init__.py

```diff
@@ -115,6 +115,11 @@ def get_inference_server_type(image_uri):
         return "mms"
     return "ts"
 
+def get_build_context():
+    return os.getenv("BUILD_CONTEXT")
```

Comment:
nit: Please replace all occurrences of `os.getenv("BUILD_CONTEXT")` with `get_build_context()`, maybe in another PR.

Reply:
Will create a separate PR